crates/holochain/src/core/workflow/app_validation_workflow/run_validation_callback_tests.rs

```diff
@@ -229,7 +229,7 @@ async fn validation_callback_awaiting_deps_agent_activity() {
                 filter_hashes.insert(delete.hashed.deletes_address.clone().clone());
                 let chain_filter = ChainFilter {
                     chain_top: delete.as_hash().clone(),
-                    filters: ChainFilters::Until(filter_hashes),
+                    stop_conditions: StopConditions::UntilHash(filter_hashes),
```

Comment:
Perhaps `limit` or `bound`? To me `stop` suggests some exit or error condition but it's really more like an intented limit on the amount of data returned.

Reply:
Changed it to `limit`.